README.rst

```diff
@@ -202,7 +251,24 @@ Updates the version and date of the most recent changelog entry.
     Arguments:
       TAG       Version number to set on the entry.
 
+Config file
```

Comment:
I'm a lazy developer and so I try to avoid useless typing or filling my memory with things that can be automated with a simple configuration file. So I added this option so that one no longer has to type the right combination of switches when tagging a release. Since the configuration file is stored in a projects root one no longer needs to remember or learn how chag tag needs to be invoked.